infra/ansible/setup/etc/hosts

```diff
@@ -1,19 +1,16 @@
 # Dynamic hosts source at /opt/ansible/inventory/gcp.yaml
-
-mongos_prod_1 ansible_ssh_host=mongos-prod-1.europe-west1-b.c.airqo-250220.internal
-
-[cfgsvrs_prod]
-cfgsvr-prod-[1:3].europe-west1-b.c.airqo-250220.internal
-
-[mongo_cluster_prod]
-mongos_prod_1
-
-[shards_prod]
-shard-prod-[1:3].europe-west1-b.c.airqo-250220.internal
-
-[mongo_cluster_prod:children]
-cfgsvrs_prod
-shards_prod
-
-[all:vars]
-ansible_python_interpreter=/usr/bin/python3
\ No newline at end of file
+all:
+  vars:
+    ansible_python_interpreter: /usr/bin/python3
+  children:
+    mongo_cluster_prod:
+      hosts:
+        mongos-prod-1.europe-west1-b.c.airqo-250220.internal
+      children:
+        cfgsvrs_prod:
+          hosts:
+            cfgsvr-prod-[1:3].europe-west1-b.c.airqo-250220.internal
+        shards_prod:
+          hosts:
+            shard-prod-[1:3].europe-west1-b.c.airqo-250220.internal
+            
```

Comment:
[change all inventories to yaml style](https://github.com/airqo-platform/AirQo-api/pull/1271/commits/21e5afa2bbb60db4c588c0eb66f1760d831978f2)